domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -13,6 +16,7 @@
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
+@JsonIgnoreProperties(value = {"totalAmountTillToday", "totalAmountTillDateOfIssue"}, allowGetters = true)
```

Comment:
Oh, did you manage to get this working in the end?

Reply:
finally yes. This let us return claim with these fields and deserialise JSON into Java object without taking these fields into account.